api/envoy/api/v2/auth/cert.proto

```diff
@@ -443,6 +443,11 @@ message DownstreamTlsContext {
     lt {seconds: 4294967296}
     gte {}
   }];
+
+  // If set to true, TLS session tickets are disabled. Setting this field to true and configuring either one of :ref:`session_ticket_keys
+  // <envoy_api_field_auth.DownstreamTlsContext.session_ticket_keys>` or :ref:`session_ticket_keys_sds_secret_config
+  // <envoy_api_field_auth.DownstreamTlsContext.session_ticket_keys_sds_secret_config>` will result in an error.
+  google.protobuf.BoolValue disable_session_tickets = 7;
```

Comment:
I'm not sure if this is the right thing to do or not, but you could put this in the oneof for session_ticket_keys_type, and that enforces in the proto that it cannot be set at the same time as a key.

Also, I think type `bool` is preferred over g.p.BoolValue`.

Reply:
I have updated the code as per feedback.

Reply:
@htuch or @mattklein123 can you verify that this is the correct way to structure the proto?

Reply:
Could we just have a `google.protobuf.Empty` message or a `bool disabled` in the `oneof` for `session_ticket_keys_type` as @ggreenway suggests?